wandb/apis/public/api.py

```diff
@@ -1157,8 +1158,14 @@ def artifact(self, name, type=None):
         if name is None:
             raise ValueError("You must specify name= to fetch an artifact.")
         entity, project, artifact_name = self._parse_artifact_path(name)
+
+        organization = ""
+        # If its an Registry artifact, the entity is an org instead
+        if is_artifact_registry_project(project):
+            parts = name.split("/")
+            organization = parts[0] if len(parts) == 3 else None
```

Comment:
Actually, this might be better than my previous suggestion
```suggestion
            if len(parts) == 3:
                organization = parts[0] // Update `organization` only if an entity name was provided, otherwise use the default that you already set above.
```